zwave_js_server/client.py

```diff
@@ -173,7 +154,8 @@ def connected(self) -> bool:
 
     async def async_send_command(self, message: Dict[str, Any]) -> dict:
         """Send a command and get a response."""
-        future: "asyncio.Future[dict]" = self._loop.create_future()
+        loop = asyncio.get_running_loop()
```

Comment:
I don't know the overhead of this compared to storing loop as instance variable

Reply:
https://github.com/python/cpython/blob/e9d4960d15c5282904cf26e469ce7cee39f634f7/Lib/asyncio/events.py#L672-L701

Reply:
It says it's implemented in c.

Reply:
`self._loop` would be a dict lookup which should be faster since just resolving `asyncio.get_running_loop` is already a dict lookup :)

Reply:
Ok, I'll switch to a loop attribute client wide.